examples/jvm/java/src/main/java/io/fission/HelloWorld.java

```diff
@@ -0,0 +1,31 @@
+package io.fission;
+
+import java.util.LinkedHashMap;
+
+import org.springframework.http.RequestEntity;
+import org.springframework.http.ResponseEntity;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+
+import io.fission.FissionFunction;
```

Comment:
Same as in Golang I would prefer `io.fission.Function` and `io.fission.Context` over these verbose names. However I am not sure if you would run into conflicts with Function from the stdlib. Might be something you could explore.